src/azure-cli/azure/cli/command_modules/acs/_params.py

```diff
@@ -397,6 +398,9 @@ def load_arguments(self, _):
         c.argument('disable_keda', action='store_true')
         c.argument('enable_vpa', action='store_true', help='enable vertical pod autoscaler for cluster')
         c.argument('disable_vpa', action='store_true', help='disable vertical pod autoscaler for cluster')
+        c.argument('enable_force_upgrade', action='store_true')
+        c.argument('disable_force_upgrade', action='store_true', validator=validate_force_upgrade_disable_and_enable_parameters)
+        c.argument('upgrade_override_until')
```

Comment:
May I ask why not add help messages in _params.py instead of _help.py?

Reply:
Some kind of AKS *tradition*, are there any downsides to this? Otherwise, is the help file just for writing samples?

Reply:
Noticed that the above two parameters have help messages in _params.py.
It is better to keep consistence for all params.